### ISSUE TITLE
false positive on  'unnessecary-pass' 

### ISSUE THREAD

**Issue by kain88-de** [PyCQA/pylint#2616]
### Steps to reproduce
1. checkout mdanalysis
2. install newest pylint
3. `cd mdanalysis/package && pylint MDAnalysis`

### Current behavior
Empty function with a pass statement fails the lint. See for [example](https://github.com/MDAnalysis/mdanalysis/blob/aaf2f9a5cd7a21a3f58cb1e7f9f71ffc71281404/package/MDAnalysis/analysis/base.py#L160)

```python
    def _prepare(self):
        """Set things up before the analysis loop begins"""
        pass
```

Corresponding failing [travis job](https://travis-ci.org/MDAnalysis/mdanalysis/jobs/460318184#L719)
and issue https://github.com/MDAnalysis/mdanalysis/issues/2148

### Expected behavior

Code is fine

### pylint --version output
https://travis-ci.org/MDAnalysis/mdanalysis/jobs/460318184#L681

This only happens with the newest version. Older versions of pylint do not report an error.

**Comment by PCManticore** [comment]
That's easy, just remove the `pass` statement. The docstring acts as a statement in this case, so `pass` is superfluous.
